ajax/libs/angular-resizable/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "angular-resizable",
+  "version": "1.2.0",
+  "description": "A directive for creating resizable containers in angular.",
+  "keywords": [
+    "angular",
+    "angular-js",
+    "angularjs",
+    "resize",
+    "resizable",
+    "directive",
+    "container",
+    "lightweight"
+  ],
+  "filename": "angular-resizable.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Reklino/angular-resizable.git"
+  },
+  "author": "Reklino",
+  "license": "MIT",
+  "homepage": "http://codepen.io/Reklino/pen/raRaXq/",
+  "npmName": "angular-resizable",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "angular-resizable*.+(js|css|map)",
+        "src/angular-resizable*.+(js|css|map)"
+      ]
+    }
```

Comment:
Try to use two filemaps here:

```js
  "npmFileMap": [
     {
       "basePath": "",
       "files": [
         "angular-resizable*"
       ]
     },
     {
       "basePath": "src",
       "files": [
         "angular-resizable*"
       ]
     }
  ]
```

So that we don't need to have a `src` folder there.